modules/ossm-rn-known-issues.adoc

```diff
@@ -34,24 +34,13 @@ These are the known issues in {SMProductName}:
 
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {SMProductName} is based on, there may be applications that are currently incompatible with {SMProductShortName}. See the linked community issue for details.
 
-* https://issues.redhat.com/browse/OSSM-1668[OSSM-1668]
-`jwksResolverCA` field is missing in `SMCP`.
-+
-If you upgrade from Service Mesh operator 2.1.3 to Service Mesh operator 2.2, then the `jwksResolverCA` field is not supported. You must use the `techPreview` `jwksResolverExtraRootCA` field to enable additional JWKS CA certificates.
 //Keep OSSM-1655 in RN, closed as "explained" error is expected.
 * https://issues.redhat.com/browse/OSSM-1655[OSSM-1655] Kiali dashboard shows error after enabling mTLS in `SMCP`.
 +
 After enabling the `spec.security.controlPlane.mtls` setting in the SMCP, the Kiali console displays the following error message `No subsets defined`.
 
 * https://issues.redhat.com/browse/OSSM-1396[OSSM-1396] If a gateway resource contains the `spec.externalIPs` setting, instead of being recreated when the `ServiceMeshControlPlane` is updated, the gateway is removed and never recreated.
 
-* https://issues.redhat.com/browse/OSSM-1211[OSSM-1211]
-Configuring Federated service meshes for failover does not work as expected.
-+
-The Istiod pilot log displays the following error: `envoy connection [C289] TLS error: 337047686:SSL routines:tls_process_server_certificate:certificate verify failed`
```

Comment:
Can we retain this error message in the Fixed Issues section?  That way customers who haven't upgraded and run into this issue will know that this release addresses this.

Reply:
I've added this back and squashed the job.